### DISCUSSION TITLE
BackBlaze b2 backup error.

### DISCUSSION THREAD

**Discussion by magic-thomas** [spatie/laravel-backup#1660] [Q&A]
laravel version : v10.8.0
spatie/laravel-backup version : 8.1.8

I get this error. 
`
Copying zip failed because: Unable to write file at location: b2Backup/2023-04-23-12-27-35.zip. Error executing "PutObject" on "https://upmainbucket.s3.us-east-005.backblazeb2.com/b2Backup/2023-04-23-12-27-35.zip"; AWS HTTP error: Client error: `PUT https://upmainbucket.s3.us-east-005.backblazeb2.com/b2Backup/2023-04-23-12-27-35.zip` resulted in a `400 Bad Request` response:`

laravel app name : b2Backup
b2 bucket name : upmainbucket 

I guess the url should be `https://upmainbucket.s3.us-east-005.backblazeb2.com/2023-04-23-12-27-35.zip` 

And I found I can not set folder name. 

Is there another way I can go around ?

**Comment by fabzster** [comment]
did you manage to resolve this by any chance?
